config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,36 @@
+Storage Bucket IAM
+==================================================
+# NAME
+  storage-bucket-iam
+# SYNOPSIS
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+# REQUIREMENTS
+  A working Config Connector cluster using a service account with `roles/storage.admin`.
+  A storage bucket managed by IAM.
```

Comment:
What does "managed by IAM" mean? Does the "Note" is used to explain it? I find it is not very clear. Does it require specific configuration of the bucket? If so, could you explain with link to examples/concept explanation/etc?

Reply:
Mark the comment as blocking merge.

Reply:
Storage buckets can be managed either by IAM or by ACL. This package will only work with the former.